draft-packages/popover/KaizenDraft/Popover/Popover.tsx

```diff
@@ -23,6 +23,9 @@ export interface Props {
   readonly singleLine?: boolean
   readonly children: React.ReactNode
   readonly boxOffset?: BoxOffset
+  // For almost all intents and purposes, you should be using a pre-defined variant.
+  // Please avoid using a custom icon unless you have a very good reason to do so.
```

Comment:
This isn't a big issue but if you use JSDoc style comments `/** ... **/` then Typescript will pick them up and include them in editor hints :)

Reply:
Aha! I knew there was a way to do that but I couldn't figure it out! Thanks!